backend/src/main/java/me/magicbudget/controller/IncomeController.java

```
@@ -50,4 +51,15 @@ public ResponseEntity<String> addIncome(@PathVariable("userid") String userId,
     }
   }
 
+  @GetMapping("/total")
+  public ResponseEntity<BigDecimal> totalIncome(@PathVariable("userid") String userId) {
+    try {
+      BigDecimal bigDecimal = incomeService.totalIncome(userId);
+      return new ResponseEntity<>(bigDecimal, HttpStatus.OK);
+    }
+    catch (Exception e) {
```

Again suppressing errors